AIDevGallery/Samples/Definitions/Models/languagemodels.modelgroup.json

```diff
@@ -199,5 +199,12 @@
         "ReadmeUrl": "https://huggingface.co/microsoft/mistral-7b-instruct-v0.2-ONNX/blob/main/README.md"
       }
     }
+  },
+  "ToolCallingLanguageModels": {
```

Comment:
I don't think this will work on the long run... Isn't a "tool calling language model", just a language model that can run tools? This would just be a property of all LanguageModels, so each definition would need to declare if they can run tools or not. We don't have a mechanism for this, so I'm not sure on the solution.
To understand concern, I think it might be easier for you to just search for "ModelType.LanguageModels". Our existing logic relies on all language models being a child of LanguageModels. All the logic for ORTGenAI, project generation, etc, relies on this.